services/apiService.ts

```diff
@@ -22,11 +21,9 @@ export const initCheck = async (data: { applicantId: string; csrf_token: string
 };
 
 export const getToken = async (applicantProperties: ApplicantProperties): Promise<ApplicantTokenPair> => {
-  const timezone = getTimezoneFromBrowser();
```

Comment:
Basically instead of getting the country based on the timezone of the browser, I directly ask the user to give the country: so the user can say his real residency country and also the getTimezoneFromBrowser() implementation is not supported by Firefox.